bulk/repo-access-api.adoc

```diff
@@ -78,8 +90,7 @@ Creates new partitions in the repository.{fn-org216}
 Each sent node is its own partition.
 Thus, we cannot send the contents (i.e. (indirect) annotations/containments) of a partition; We can send them in a later <<store>> call.
 We also MUST NOT mention any annotation/containment node ids in the partition nodes, as they cannot be part of the same request, and we don't allow moving nodes in this operation.
-
-We MAY send properties and references #TODO correct?#{fn-org225}
+We MAY send properties and references{fn-org225}
 
 Each partition node id MUST NOT exist in the repository, and the sending client MUST use node ids allocated to it via <<ids>>.
```

Comment:
This would be the only case where the node ids MUST be allocated through <<ids>>.  As with the other node ids, I think this should be optional.